src/js/main.js

```diff
@@ -121,7 +123,7 @@ define(function (require, exports) {
             });
 
         var renderPromise = new Promise(function (resolve) {
-            React.render(new Main(props), window.document.body, function () {
+            ReactDOM.render(new Main(props), window.document.getElementById("app"), function () {
```

Comment:
Hmm, I guess this is why? But, you should still just use the class instead of adding a new ID.